examples/deployment/docker/db_client/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.9
+FROM golang:1.10
```

Comment:
Do we want to just use `golang:1` to keep track with the latest revision of Go?

Reply:
We could, but then the Docker images might be built with a newer version of Go than Travis uses.

Reply:
Might be better to keep it explicit for the moment to avoid surprises.